Gemfile

```diff
@@ -1,4 +1,6 @@
-source "https://rubygems.org" do
-  gem "cocoapods", '~> 1.8.0'
-  gem "xcpretty"
-end
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+
+gem 'cocoapods', '~> 1.10.0'
```

Comment:
Should relax this a bit and go for `~> 1.10`? Not a requirement for this PR, though, just a question.

Reply:
Yep, though I disagree – it should be a requirement for this PR please!!! 😄

Reply:
I'll change it!